package.json

```diff
@@ -35,7 +35,6 @@
     "@angular/http": "^4.1.0"
   },
   "dependencies": {
-    "moment": "^2.9.0"
```

Comment:
We don't need moment anymore? How will we give human-readable relative times like '2 minutes ago' or '3 hours ago'?